client/src/components/OpportunityView.js

```diff
@@ -16,23 +16,14 @@ import { Link } from "react-router-dom";
 import { getLoggedInUserData } from "../utils/storage";
 
 export default class OpportunityView extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      opportunityId: null,
-      opportunity: [],
-      skills: [],
-      isLoading: true,
-      isEditing: false
-    };
-  }
-  //Get data and Id
-  componentWillMount() {
-    const { pathname } = window.location;
-    this.setState({
-      opportunityId: pathname && pathname.replace("/opportunities/", "")
-    });
-  }
+  state = {
```

Comment:
Does the page not break if you refresh it now?

Reply:
also please pull upstream

Reply:
no it doesn't it is working well and i did pull upsteram and solved the conflicts